### PR TITLE
Updated IAM Policy and added default null values for autoscaler vars

### PR DESCRIPTION
**Pull request by eminalemdar**
## Description of the change

I've changed the IAM Policy for the Lambda Autoscaler function. Also, I've added default `null` values for the `spacelift_api_key_*` variables.

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue);
- [ ] New feature (non-breaking change that adds functionality);
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected);
- [ ] Documentation (a documentation or example fix not affecting the infrastructure managed by this module);

### Development

- [x] All necessary variables have been defined, with defaults if applicable;
- [x] The code is formatted properly;

### Code review

- [x] The module version is bumped accordingly;
- [x] Spacelift tests are passing;
- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached;
- [x] This pull request is no longer marked as "draft";
- [x] Reviewers have been assigned;
- [ ] Changes have been reviewed by at least one other engineer;

### CODE REVIEW

File: .spacelift/config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 version: 1
-module_version: 1.8.0
+module_version: 1.9.0
```
Comment (jmfontaine) [inline .spacelift/config.yml:2]: I would consider that a fix for version 1.8.0. It does not introduce any new features.

```suggestion
module_version: 1.8.1
```

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ terraform {
 }
 
 module "my_workerpool" {
-  source = "github.com/spacelift-io/terraform-aws-spacelift-workerpool-on-ec2?ref=v1.8.0"
+  source = "github.com/spacelift-io/terraform-aws-spacelift-workerpool-on-ec2?ref=v1.9.0"
```
Comment (jmfontaine) [inline README.md:22]:
```suggestion
  source = "github.com/spacelift-io/terraform-aws-spacelift-workerpool-on-ec2?ref=v1.8.1"
```

File: .spacelift/config.yml (hunk 1 above)
Comment (eminalemdar) [reply .spacelift/config.yml]: Updated

File: README.md (hunk 2 above)
Comment (eminalemdar) [reply README.md]: Updated

**Review by jmfontaine — APPROVED** [review]

**Pull request merged**